content/pt/docs/reference/kubectl/cheatsheet.md

```diff
@@ -0,0 +1,387 @@
+    - "1000"
+EOF
+
+# Crie um segredo com várias chaves
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mysecret
+type: Opaque
+data:
+  password: $(echo -n "s33msi4" | base64 -w0)
+  username: $(echo -n "jane" | base64 -w0)
+EOF
+
+```
+
+## Visualizando e localizando recursos
+
+```bash
+# Obter comandos com saída simples
+kubectl get services                          # Listar todos os serviços do namespace
+kubectl get pods --all-namespaces             # Listar todos os pods em todos namespaces
+kubectl get pods -o wide                      # Listar todos os pods no atual namespace, com mais detalhes
+kubectl get deployment my-dep                 # Listar uma implantação específica
+kubectl get pods                              # Listar todos os pods no namespace
+kubectl get pod my-pod -o yaml                # Obter o YAML de um pod
+kubectl get pod my-pod -o yaml --export       # Obtenha o YAML de um pod sem informações específicas do cluster
+
+# Descrever comandos com saída detalhada
+kubectl describe nodes my-node
+kubectl describe pods my-pod
+
+# Listar serviços classificados por nome
+kubectl get services --sort-by=.metadata.name
+
+# Listar pods classificados por contagem de reinicializações
+kubectl get pods --sort-by='.status.containerStatuses[0].restartCount'
+
+# Lista PersistentVolumes classificado por capacidade
+kubectl get pv --sort-by=.spec.capacity.storage
+
+# Obtenha a versão da label de todos os pods com a label app=cassandra
+kubectl get pods --selector=app=cassandra -o \
+  jsonpath='{.items[*].metadata.labels.version}'
+
+# Obter todos os nós dos workers  (use um seletor para excluir resultados que possuem um rótulo
+# nomeado 'node-role.kubernetes.io/master')
+kubectl get node --selector='!node-role.kubernetes.io/master'
+
+# Obter todos os pods em execução no espaço para nome
+kubectl get pods --field-selector=status.phase=Running
+
+# Obter ExternalIPs de todos os nós
+kubectl get nodes -o jsonpath='{.items[*].status.addresses[?(@.type=="ExternalIP")].address}'
+
+# Listar nomes de pods pertencentes a um particular RC
+# O comando "jq" é útil para transformações que são muito complexas para jsonpath, pode ser encontrado em https://stedolan.github.io/jq/
+sel=${$(kubectl get rc my-rc --output=json | jq -j '.spec.selector | to_entries | .[] | "\(.key)=\(.value),"')%?}
+echo $(kubectl get pods --selector=$sel --output=jsonpath={.items..metadata.name})
+
+# Mostrar marcadores para todos os pods(ou qualquer outro objeto Kubernetes que suporte rotulagem)
+kubectl get pods --show-labels
+
+# Verifique quais nós estão prontos
+JSONPATH='{range .items[*]}{@.metadata.name}:{range @.status.conditions[*]}{@.type}={@.status};{end}{end}' \
+ && kubectl get nodes -o jsonpath="$JSONPATH" | grep "Ready=True"
+
+# Listar todos os segredos atualmente em uso por um pod
+kubectl get pods -o json | jq '.items[].spec.containers[].env[]?.valueFrom.secretKeyRef.name' | grep -v null | sort | uniq
+
```

Comment:
Looks like there are some examples that are missing. Can you please check.

Reply:
I didn't find any examples missing. Could you please explain?

Reply:
I checked the translation against https://kubernetes.io/docs/reference/kubectl/cheatsheet/. 
I believe the missing part is this:
```
# List all containerIDs of initContainer of all pods
# Helpful when cleaning up stopped containers, while avoiding removal of initContainers.
kubectl get pods --all-namespaces -o jsonpath='{range .items[*].status.initContainerStatuses[*]}{.containerID}{"\n"}{end}' | cut -d/ -f3
```

Reply:
Good catch @devlware ! just updated,  thanks